### PR TITLE
Add working Shotwell. See README.md

### PR DESCRIPTION
**Pull request by LyzardKing**
It's not perfect, but it managed to start (with --devmode) and import all my photos without any problem.
Without the --devmode option it wouldn't even start.

### CODE REVIEW

**Comment by tsimonq2** [comment]
Doesn't work with `snapcraft cleanbuild`: http://paste.ubuntu.com/17563401/

**Comment by LyzardKing** [comment]
The problem is a new commit added roughly 4 hours ago to shotwell...
For now I added a source-tag to the latest release (still newer than the one in the ubuntu repos...)
I'll revisit it, as far as I can tell the new commit added a separate makefile for the plugins...
For now this works...and it has a .desktop file.
@tsimonq2 Let me know if you test this!

**Comment by tsimonq2** [comment]
@LyzardKing Builds fine now! Thanks!

**Comment by wandrewkeech** [comment]
Terrific! Works when `snap install`ed with `--devmode`. On normal install, thrown error:
```
/snap/shotwell/x1/bin/gtk-launch: line 111: 142916 Bad system call         cp -a $SNAP/usr/share/mime $XDG_DATA_HOME

(process:142901): Gtk-WARNING **: Locale not supported by C library.
	Using the fallback 'C' locale.
Gtk-Message: Failed to load module "unity-gtk-module"
Gtk-Message: Failed to load module "canberra-gtk-module"
Gtk-Message: Failed to load module "canberra-gtk-module"
```
and 
```L 143493 2016-06-19 20:35:23 [CRT] GSettingsEngine.vala:446: GSettingsConfigurationEngine: error: schema 'org.yorba.shotwell.plugins.enable-state' is not installed
L 143493 2016-06-19 20:35:23 [CRT] ConfigurationInterfaces.vala:371: configuration engine 'GSettings' reports ENGINE_ERROR: schema 'org.yorba.shotwell.preferences.ui' is not installed
```

The `gtk` errors are thrown despite updating `stage-packages` to include both `unity-gtk-modules-common` and 'libcanberra-gtk-module`

**Comment by LyzardKing** [comment]
Yes, I tried adding the gtk modules, but nothing changed.
I wrote in the readme that for now it works only with devmode...
I'll look into it, but at least it works ;)

File: shotwell/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+Working features:
+ - Photo import from folder
+
+Known issues:
+  - Only starts if installed with --devmode
+
+TODO:
+ -
+
```
Comment (dholbach) [inline shotwell/README.md]: Maybe add that it should be worked out why this doesn't work without `--devmode`? Can you maybe bring it up as a question on snapcraft@lists.snapcraft.io?

Comment (LyzardKing) [inline shotwell/README.md]: I added the error I get without --devmode

**Comment by dholbach** [comment]
Nice work!

**Pull request merged**